spec.html

```diff
@@ -8258,7 +8258,6 @@
 
       <emu-clause id="sec-set-immutable-prototype">
         <h1>SetImmutablePrototype ( _O_, _V_ )</h1>
-        <!-- This is used by host environment specs, even though it only has one consumer in this spec; don't inline it -->
```

Comment:
SetImmutablePrototype is used in 9.4.6.1 and 9.4.7.1. And we don't control which abstract operations are used by external parties, so we can't really mark every operation that's used externally. Allen's proposal in #398 seems more doable than having an incomplete coverage of operations which may or may not be used somewhere else.